frontend/src/exams/exams.vue

```diff
@@ -8,15 +8,16 @@
 </template>
 
 <script>
+  import { mapActions } from 'vuex'
   import ExamInventoryTable from './exam-inventory-table'
 
   export default {
     name: "Exams",
-    components: { ExamInventoryTable }
+    components: { ExamInventoryTable },
   }
 </script>
 
-<style scoped>
+<style scoped>[]
```

Comment:
What's with the `[]`?

Reply:
hmmm....  you'd think that woud have  thrown an error of some sort.  typo.